Doc/library/json.rst

```diff
@@ -299,8 +300,9 @@ Basic Usage
 
    The other arguments have the same meaning as in :func:`load`.
 
-   If the data being deserialized is not a valid JSON document, a
-   :exc:`JSONDecodeError` will be raised.
+   If the data is not a valid JSON document, a :exc:`JSONDecodeError` will be
+   raised. If the binary file does not contain UTF-8, UTF-16 or UTF-32 encoded
```

Comment:
There is no file here. In any case the wording differs from the suggested by @rhettinger, so I leave this on him.

Reply:
This seems fine to me.